gbpservice/nfp/orchestrator/config_drivers/heat_driver.py

```diff
@@ -246,6 +246,20 @@ def loadbalancer_post_stack_create(self, network_function_details):
                     if port_info['port_model'] != nfp_constants.GBP_PORT:
                         return
 
+    def _post_stack_create(self, nfp_context):
+        service_details = self.get_service_details_from_nfp_context(
+            nfp_context)
+        service_details = service_details['service_details']
+        service_type = service_details['service_type']
+
+        if service_type in [pconst.LOADBALANCER]:
+            logging_context = nfp_logging.get_logging_context()
+            auth_token = logging_context['auth_token']
```

Comment:
this auth_token is tenant_token, we are passing this additional token now from nfp node driver. And in orchestrator, we had to store this token in execution thread context so that we need not pass it explicitly in each and every function call and thus avoiding function signature change. We have already defined logging context per thread, which is a dict, reusing it to store tenant_token too.